### DISCUSSION TITLE
Archive pages look different when build offline/online

### DISCUSSION THREAD

**Discussion by Natenom** [razonyang/hugo-theme-bootstrap#742] [Q&A] [answered]
Hi, I am not sure if this is a bug or wrong configuration.

When I build my blog offline on my pc, the archive page looks good and I can open a years list of all blog posts grouped by month.

When I build it on my server with the same extended hugo version (103.0 currently) the archive pages look all the same. Sub pages per year look the same as the top archive page. See https://blog.natenom.com/archiv/2022/

Am I missing something? I am sure it worked some time ago and tried older checkouts from the theme but it didn't work.

**Comment by razonyang** [comment]
Could you please check the `archive.basePath` parameter both default and production environment?
Since you're using custom archives path(`archiv`), you'll need to change `archive.basePath` to `archiv`.

**Comment by Natenom** [comment, marked as answer]
Sorry for the noise. It was related to the missing/wrong spelled `baseURL`. (https://github.com/razonyang/hugo-theme-bootstrap/issues/741#issuecomment-1257135766)
